apps-rendering/src/components/Series/GallerySeries.tsx

```diff
@@ -28,8 +28,8 @@ const styles = css`
 `;
 
 const linkStyles = (format: ArticleFormat): SerializedStyles => css`
-	color: ${text.seriesTitle(format)};
-	background-color: ${background.series(format)};
+	background-color: ${text.seriesTitle(format)};
+	color: ${background.series(format)};
```

Comment:
Should these be the other way round in the palette? That way we don't end up using `background` for `color` and `text` for `background-color`?